src/app/components/elements/inputs/UserContextAccountInput.tsx

```diff
@@ -171,8 +170,8 @@ export function UserContextAccountInput({
             userContexts.findIndex((p) => p.stage === STAGE.ALL && p.examBoard === EXAM_BOARD.ALL) === -1;
 
           return (
-            <RS.FormGroup key={index}>
-              <RS.Row>
+            <FormGroup key={index}>
```

Comment:
This issue was not introduced in this ticket, however sonarCloud is flagging it so it's worth removing the use of `index` for the key here!

Reply:
To fix this issue we need to create unique IDs for the userContext. 
You will then need to change src/IsaacApiTypesAutoGenerated.tsx file adding : `id: Key | null | undefined;`
I have found out that the website does not like this, probably because it is looking for it in the API. 
I think we need a new ticket for this. It is not as straight forward.

Reply:
It shouldn't be necessary to add IDs to the userContexts, you could set the key to something like `${userContext.stage}-${userContext.examBoard}` and this would always be a unique ID (because once you've selected GCSE & AQA as a user context, you can't select it again but you might have GCSE & OCR). So userContext.stage or userContext.examBoard alone wouldn't be a good unique identifier but I'd expect the combination to work.

Reply:
Added the change in now 👍